todo_app/flask_config.py

```diff
@@ -8,3 +8,5 @@ def __init__(self):
         if not self.SECRET_KEY:
             raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
         self.LOGIN_DISABLED = os.getenv('LOGIN_DISABLED') == 'True'
+        self.LOG_LEVEL = os.getenv('LOG_LEVEL')
```

Comment:
Currently your app errors if this isn't set - instead I'd be tempted to set a default level here so that the system doesn't error on a missing setting for this value, e.g.
```suggestion
        self.LOG_LEVEL = os.getenv('LOG_LEVEL', 'INFO')
```